editoast/openapi.yaml

```diff
@@ -8527,7 +8527,7 @@ components:
       description: |-
         A search result item for a query with `object = "signal"`
 
-        **IMPORTANT**: Please note that any modification to this struct should be reflected in [crate::modelsv2::infra::Infra::clone]
+        **IMPORTANT**: Please note that any modification to this struct should be reflected in [crate::models::infra::Infra::clone]
```

Comment:
Should these comments be shown in the OpenAPI? Maybe we can change some `///` into some `//`?

Reply:
I'm waiting for https://github.com/OpenRailAssociation/osrd/pull/8526 to be merged so that @younesschrifi doesn't have to deal with the conflicts. I'll probably have to redo the last commit entirely, I'll change the `///` at the same time.

Reply:
Thank you ! #8526 is merged 🙂

Reply:
Ill wait for https://github.com/OpenRailAssociation/osrd/pull/8712 as well.